### PR TITLE
Update UserGuide and find commands

### PR DESCRIPTION
**Pull request by tsumian**
Delete "food" and "owner" from UG since "find" will already display
all relevant details.
Add MESSAGE_PETS_FOUND to Messages.java to display "Found
___ pets similar to keywords" instead of just "___ pets listed!". Edit
relevant test cases with expected messages.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -118,12 +96,10 @@ Format: `exit`
...
-| **Time**   | `time id `<br> e.g.,`pickup 3 0900 1200` (where 3 is the id of the pet in the system)                                                          |
-| **Food**   | `food id `<br> e.g., `food 3` (where 3 is the id of the pet in the system)                                                                     |
-| **Owner**  | `owner o/OWNER_NAME` <br> e.g., `owner o/Jeff Lin`                                                                                             |
-| **Exit**   | `exit`                                                                                                                                         |
+| Action     | Format, Examples                                                                                                                              |
+|------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add**    | `add n/NAME_OF_PET o/OWNER_NAME p/PHONE_NUMBER a/ADDRESS` <br> e.g., `add n/Peepee o/Peter p/98648252 a/13 Computing Drive, Singapore 117417` |
```
Comment (soloplxya) [inline docs/UserGuide.md:101]: Edits look good. Possible to add the diet command here as well: 
` diet id d/remark`

**Review by soloplxya — COMMENTED** [review]
Looks good! LGTM!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -53,36 +53,14 @@ Examples:
 
 ### Find pet Id: `find`
```
Comment (Dineshraj555) [inline docs/UserGuide.md:54]: I think it will be better to rename **Find pet Id:** to a more appropriate name. Maybe **Find pet details:** ?

**Review by Dineshraj555 — COMMENTED** [review]
LGTM. Just one minor naming issue to rectify.

**Pull request merged**